qutebrowser/completion/models/miscmodels.py

```diff
@@ -47,6 +47,31 @@ def helptopic(*, info):
     return model
 
 
+def suggest(*args, **kwargs):
+    """A CompletionModel filled with already typed suggestions."""
+    model = None
+
+    try:
+        cols = int(args[1])
+        col_size = int(100 / cols)
+        sep_sug = args[2]
+        sep_col = args[3]
+        model = completionmodel.CompletionModel(
+            column_widths=tuple(col_size for i in range(cols)))
+        raw = (args[4][1:-1]
+               if any(args[4].startswith(i) for i in ('"', "'"))
+               else args[4])
+        suggestions = (i.split(sep_col) for i in raw.split(sep_sug))
+
+        model.add_category(listcategory.ListCategory(
+            "Suggestions", suggestions))
+    except Exception as e:
```

Comment:
We should be more specific about the exception/message here. For example, if the arg count isn't right, then specifically catch and report that.